src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String COMMA = ",";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String REJEX = "\\W+";
+    private int sumOfSupply = 0;
+    private int sumOfBuy = 0;
+    private int result;
+    private String fileName;
```

Comment:
this is not a very good practice to store state in service classes

so let's move these variables (sumOfSupply, sumOfBuy, result, fileName) into method